pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Does your code effect "Classic" (if applicable):
+
+<!--- Because the compiled styles will automatically publish to CLASSIC IN PRODUCTION please be sure your changes won't effect "classic". -->
+
+### If code effected "Classic" what is effected and how is it being handled:
```

Comment:
This is really, really specific to non-open source efforts at TeamSnap. Can we remove this?

Reply:
Very good point. Removed